src/app/components/available-in-other-languages/available-in-other-languages.component.html

```diff
@@ -0,0 +1,13 @@
+<ng-container *ngIf="otherLanguages.length">
+  <div class="also-available">
```

Comment:
This doesn't look bad at all. How about a `<ul>`? We're anticipating multiple languages.

Reply:
I've added a <ul> for semantic, I guess. It still looks the same.

Reply:
I was suggesting we use the standard visual layout of ul as well.


This content is available in the these languages:
* Russian
* French
* etc

Reply:
The prompt is shown on top of the article, I don't think this would look good.

Reply:
👌 there's a conflict. happy to merge after that is fixed.